RecoTracker/DisplacedRegionalTracking/plugins/DisplacedRegionSeedingVertexProducer.cc

```diff
@@ -93,16 +94,24 @@ void DisplacedRegionSeedingVertexProducer::produce(edm::StreamID streamID,
   const auto &trackClusters = event.get(trackClustersToken_);
 
   // Initialize distances.
-  list<DisplacedVertexCluster> pseudoROIs;
-  list<Distance> distances;
+  vector<DisplacedVertexCluster> pseudoROIs;
```

Comment:
NOTE: though not part of the changes, the `DsiplacedVertexCluster` class looks like it has an excessive amount of padding since its member data order starts with a `bool`

https://github.com/cms-sw/cmssw/blob/83deecd45f65caa502f93b62e417dfd8720948a7/RecoTracker/DisplacedRegionalTracking/plugins/DisplacedVertexCluster.h#L58-L63

It seems to me like the `bool valid_` could be removed and instead, use an empty `constituents_` to denote an invalid object. (i.e. call to `setInvalid()` would empty the vector and `bool valid const` would just call `constituents_.empty()`

Reply:
i'll leave this one to the experts..